lib/tracker_api/resources/story.rb

```diff
@@ -44,6 +44,10 @@ def tasks(params = {})
           @tasks = Endpoints::Tasks.new(client).get(project_id, id, params)
         end
       end
+
+      def update(story, params)
```

Comment:
@wulffeld seems a little odd to pass the story into the `update` as it is a method on story.

The ideal interface for update would be something like this.
```
story.name = "New awesome name"
story.save
story.attributes = { name: 'More awesome name', description: 'Do something like this' }
story.save
```

The attributes mass assignment comes from Virtus.

You will probably need to decide in save if you should call put or post (update or create). I think the existence of the id should tell you that.